### PR TITLE
Disable automatic insertion into 15.9

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

**Comment by agocke** [comment]
@jasonmalinowski Please take a look and make sure I'm right about this

**Comment by jasonmalinowski** [comment]
@agocke Wouldn't we retarget to the newer branch or whatever it is? @dpoeschl implemented a "stop trying to insert this particular build" feature which can be used to shut up duplicates of things we don't want anymore.

File: src/RoslynInsertionTool/scripts/VS-Insertion.ps1 [hunk 1]
```diff
@@ -59,10 +59,10 @@ Do-Insertion -component "F#"                -queueName "FSharp-Signed"         -
...
 
 # Dev15.9 Preview 4
-Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "dev15.9.x-vs-deps" -toBranch "lab/d15.9stg" -insertToolset "false" -insertDevdiv "false" -updatecorextlibraries "true"
-Do-Insertion -component "Live Unit Testing" -queueName "TestImpact-Signed"     -fromBranch "dev15.9.x"         -toBranch "lab/d15.9stg" -insertCore "false"    -insertDevdiv "false" -queueValidation "true"
-Do-Insertion -component "Project System"    -queueName "DotNet-Project-System" -fromBranch "dev15.9.x"         -toBranch "lab/d15.9stg" -insertCore "false"    -insertDevdiv "false" -queueValidation "true" -updateassemblyversions "false"
-Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "dev15.9.x"         -toBranch "lab/d15.9stg" -insertCore "true"                           -queueValidation "true" -dropPath "server"
+Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "dev15.9.x-vs-deps" -toBranch "rel/d15.9" -insertToolset "false" -insertDevdiv "false" -updatecorextlibraries "true"
```
Comment (jasonmalinowski) [inline src/RoslynInsertionTool/scripts/VS-Insertion.ps1]: Seems odd this one doesn't have queueValidation?

Comment (agocke) [reply src/RoslynInsertionTool/scripts/VS-Insertion.ps1]: That is weird.

**Pull request merged**